tests/transformer/idx_list_image_latest_test.py

```diff
@@ -14,3 +14,18 @@ def test_transformer_idx_list_image_latest():
         "diff ${PWD}/tests/transformer/testdata/expected/idx/list/sort-by-date/0"
         " ${PWD}/tests/transformer/testdata/output/idx/list/sort-by-date/0"
     )
+
+    assert 0 == os.system(
+        "diff ${PWD}/tests/transformer/testdata/expected/idx/list/sort-by-date-aws/0"
+        " ${PWD}/tests/transformer/testdata/output/idx/list/sort-by-date-aws/0"
+    )
+
+    assert 0 == os.system(
+        "diff ${PWD}/tests/transformer/testdata/expected/idx/list/sort-by-date-google/0"
+        " ${PWD}/tests/transformer/testdata/output/idx/list/sort-by-date-google/0"
+    )
+
+    assert 0 == os.system(
+        "diff ${PWD}/tests/transformer/testdata/expected/idx/list/sort-by-date-azure/0"
```

Comment:
@major I will change this to the same style as you made it for the google test - I promise :v::grin:

I created a [ticket](https://github.com/redhatcloudx/cloud-image-directory/issues/441) for that :hugs:

Reply:
Woot!